.github/workflows/ci.yml

```
@@ -1,8 +1,8 @@
 name: '@ngworker/lumberjack CI'
 
 env:
-  ANGULAR_LATEST: 11.1.x
-  NODE_LATEST: 12.x
+  ANGULAR_LATEST_STABLE: 11.2.x
```

1) Add this comment to make it more clear.
```suggestion
  # must match the version in `package.json`
  ANGULAR_LATEST_STABLE: 11.2.x
```

---

2. Either update the dependencies in `package.json` or roll back this chagne.